index.html

```diff
@@ -1,13 +1,15 @@
 <!doctype html>
-<html>
+
+<html lang="en">
 <head>
-	<meta charset="utf-8">
-	<title>HTML & CSS Training</title>
-	<meta name="description" content="training">
-	<meta name="author" content="Wolox">
-	<link rel="stylesheet" type="text/css" href="application.css">
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" href="application.css" type="text/css">
 </head>
 <body>
-	<h1 class="title">Hello World</h1>
+  <h1 class="text-big page-title">Front-End Tools</h1>
```

Comment:
Use only `page-title` here and inside `page-title` class in css add the font-size importing @fonts